site/profile/files/sample_website/css/main.css

```diff
@@ -1,18 +1,23 @@
-body { margin: 60px auto; width: 70%; background:black url(../img/logo.png) no-repeat left top;
+body {
+  margin-top: 200px;
+  margin-left: 60px;
+  width: 70%;
+  font-family: 'Helvetica', 'Arial', 'Sans-Serif';
+  background:black url(../img/logo.png) no-repeat left top;
```

Comment:
@maju6406 I just consolidated the font-family in body so it affects all child elements.